examples/glut_example.nim

```diff
@@ -88,7 +88,8 @@ proc reshape(width: GLsizei, height: GLsizei) {.cdecl.} =
   # Enable perspective projection with fovy, aspect, zNear and zFar
   gluPerspective(45.0, width / height, 0.1, 100.0)
 
-glutInit()
+var x: cint = 0
```

Comment:
thank you for creating this, but seeing as this is an example let's make it more readable. Rename this to ``argc``.